contracts/Core/StakeManager.sol

```diff
@@ -345,6 +347,14 @@ contract StakeManager is Initializable, ACL, StakeStorage, StateManager, Pause {
         return stakers[stakerId].epochFirstStaked;
     }
 
+    /// @return isStakerActive : Activity < Grace
+    function isStakerActive(uint32 stakerId, uint32 epoch) public view returns (bool) {
+        uint32 epochLastRevealed = voteManager.getEpochLastRevealed(stakerId);
+        uint32 inactiveEpochs = (epoch - epochLastRevealed == 0) ? 0 : epoch - epochLastRevealed - 1;
+        if (inactiveEpochs < parameters.gracePeriod()) return true;
+        return false;
```

Comment:
the three lines above can be combined for more efficient code

return ( (epoch - epochLastRevealed) >= parameters.gracePeriod());

Reply:
done https://github.com/razor-network/contracts/pull/380/commits/28f2a75119b2175b53a2f9dad03ac5fea363c509